workflow/doc.go

```diff
@@ -384,31 +392,6 @@ workflow function should terminate by returning the special ContinueAsNewError e
 
 For a complete example implementing this pattern please refer to the Cron example.
 
-SideEffect API
```

Comment:
Looks like there's a lot of useful information here that's not found in our docs (e.g. that a side-effect can be executed more than once).